### PR TITLE
changes added

### PR DESCRIPTION
**Pull request by Anjutechversant22**

### CODE REVIEW

**Comment by Anjutechversant22** [comment]
change?

File: user_management/templates/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>welcome</h1>
```
Comment (Anjutechversant22) [inline user_management/templates/index.html:1]: hgfhdf

Comment (Anjutechversant22) [reply user_management/templates/index.html:1]: fghdfh

**Comment by Harikrishnanvc** [comment]
closing

**Pull request closed without merging**